cdc/sinkv2/eventsink/txn/backend.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package txn
+
+import (
+	"github.com/pingcap/tiflow/cdc/sinkv2/eventsink"
+	"github.com/pingcap/tiflow/pkg/notify"
+)
+
+type backend interface {
+	onTxnEvent(*eventsink.TxnCallbackableEvent) error
+	onNotify() error
+	notifier() *notify.Receiver
```

Comment:
Why are these methods private?

Reply:
All backends will be put in this package, and they won't be used out from the package.